### ISSUE TITLE
util.rds documentation missing for JS resolvers

### ISSUE THREAD

**Issue by sashee** [aws/aws-appsync-community#264]
The VTL documentation shows a $util.rds.toJsonObject function:

![image](https://user-images.githubusercontent.com/82075/219857507-7b3756fa-d1ef-4b3a-8293-6a2d6cd6a8da.png)

The same is available for the JS resolvers (from the @aws-appsync/utils npm package https://www.npmjs.com/package/@aws-appsync/utils?activeTab=explore):

![image](https://user-images.githubusercontent.com/82075/219857518-08d3d807-67dd-4d53-870b-1ae98cc5f472.png)

But the documentation page for it is missing:

![image](https://user-images.githubusercontent.com/82075/219857537-bb669237-2c08-487f-9d4b-99d8c4d191d1.png)

**Comment by onlybakam** [comment]
Hey @sashee , are you available to connect? I'm interested to learn more about your experience with this relational database Data source

**Comment by sashee** [comment]
Hey @onlybakam , sure, would be glad to help!

**Comment by onlybakam** [comment]
awesome. can you DM me on twitter (`@bricepelle`) ?

**Comment by sashee** [comment]
I'm not using Twitter that much but it seems like it does not let me DM you. Could you send a mail to tamas.sallai@advancedweb.hu instead?

**Comment by onlybakam** [comment]
tracking #263 here as well.

**Comment by alextriaca** [comment]
+1 on this issue. We make extensive use of AppSync -> RDS to service our frontend. We're in the process of moving from VTL to Javascript and this would be invaluable.

**Comment by onlybakam** [comment]
Hey @alextriaca, are you available to connect? would like to learn more about your use case.

**Comment by magus424** [comment]
+1 as well, looking into appsync and we're going to be putting data into RDS soon but the lack of docs in the JS resolver info is quite annoying.

**Comment by magus424** [comment]
or are we meant to be using Lambda stuff in between AppSync and RDS now with JS resolvers? sort of getting that impression from the docs

**Comment by monscamus** [comment]
Yep, we too are migrating from VTL. Would be nice to understand more of the undocumented quirks of utils/rds etc.

**Comment by onlybakam** [comment]
we've updated the docs for JS resolvers here: https://docs.aws.amazon.com/appsync/latest/devguide/resolver-reference-rds-js.html and here: https://docs.aws.amazon.com/appsync/latest/devguide/built-in-modules-js.html#built-in-rds-modules.

Let us know if you have any feedback.

**Comment by sashee** [comment]
Thanks @onlybakam , the documentation looks nice! Also great to see SQL tagged templates, it is way superior to the old API.
